frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -78,8 +78,11 @@ private void processIndentationElement(PsiElement element, AnnotationHolder hold
 
     var thisIndentationText = element.getText();
 
-    if (thisIndentationText.chars().filter(c -> c != indentationParameters.indentationCharacter).count() > 0) {
-      holder.newAnnotation(HighlightSeverity.ERROR, "Indentation character does not match the first indented line")
+    var wrongIndentChar = thisIndentationText.chars().filter(c -> c != indentationParameters.indentationCharacter).findFirst();
```

Comment:
Hmm actually what happens when the file is empty or there's no indent on any line?

Reply:
Nothing special - this piece of code will even not being executed